changelog/v2.11.3/changelog.md

```diff
@@ -0,0 +1,29 @@
+<p align="center">
+  <img src="http://www.mikroe.com/img/designs/beta/logo_small.png?raw=true" alt="MikroElektronika"/>
+</p>
+
+---
+
+**[BACK TO MAIN FILE](../../changelog.md)**
+
+---
+
+# `v2.11.3`
+
++ released: TBD
```

Comment:
Release is planned for October 1-st this year. Update the year accordingly here.

Reply:
Updated date accordingly.